internal/protoschema/protoschema.go

```diff
@@ -14,7 +14,15 @@
 
 package protoschema
 
-const (
-	// Version is the current version of protoschema which is shared by all plugins.
-	Version = "0.2.0-dev"
+import (
+	"runtime/debug"
+	"strings"
 )
+
+func Version() string {
```

Comment:
We don't want to do this - we want the version to be static and part of the build, same as our other OSS repos. Also no Godoc

@mfridman just spent a good amount of time getting this OSS-ready, check in with him before merging

Reply:
Talked to @Alfus offline - @mfridman let's talk about this Monday, I'm against this style

Reply:
I think pre-Go modules the 2-step commit made sense. But with Go modules and >=go1.12+ this has become the most ergonomic way to handle versioning.

Maybe it would be worth chatting internally if this can be an acceptable way of propagating a version?

Reply:
Got it - let's chat Monday you me @Alfus and make sure we're on same page - if this is the way forward, I'd want to start making it the OSS standard across our repos (can convert over time)

Reply:
One of the things is id worry what this means for ie dev builds, but yea lets chat Monday

Reply:
Sure!